src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -344,7 +344,7 @@ internal static bool TryExpandDateTimeExcelFormatSpecifiersToStringValue(IRConte
                 case "'longtime24'":
                 case "'longtime'":
                 case "'longdate'":
-                    result = new StringValue(irContext, dateTime.ToString(ExpandDateTimeFormatSpecifiers(format, culture)));
+                    result = new StringValue(irContext, dateTime.ToString(ExpandDateTimeFormatSpecifiers(format, culture), culture));
```

Comment:
nit:  can we extra this part to a local (just breaking up the long expression). 
var formatStr = ExpandDateTimeFormatSpecifiers(format, culture);

That also makes it clearer that the ToString() was missing a culture parameter.